calico-cloud/multicluster/kubeconfig.mdx

```diff
@@ -219,46 +229,137 @@ We'll now create the RemoteClusterConfigurations that establish synchronization
 
 1. Repeat the above steps, switching cluster A and cluster B.
 
 
 After completing the above steps for all cluster pairs in the cluster mesh, your clusters should now be ready to utilize remote-identity-aware policy and federated services, along with multi-cluster networking if requirements were met.
 
-:::note
+</TabItem>
+<TabItem value="Unencapsulated Routing" label="Unencapsulated Routing">
+
+In this setup, the cluster mesh will rely on the [underlying network](#other-networking-configurations) to meet the pod IP routability requirement.
+
+**For each pair** of clusters in the cluster mesh (e.g. cluster A and cluster B):
+
+1. In cluster A, create a secret that contains the kubeconfig for cluster B:
+
+   Determine the namespace (`<secret-namespace>`) for the secret to replace in all steps.
+   The simplest method to create a secret for a remote cluster is to use the `kubectl` command because it correctly encodes the data and formats the file.
+   ```bash
+   kubectl create secret generic remote-cluster-secret-name -n <secret-namespace> \
+      --from-literal=datastoreType=kubernetes \
+      --from-file=kubeconfig=<kubeconfig file>
+   ```
+
+1. If RBAC is enabled in cluster A, create a Role and RoleBinding that {{prodname}} will use to access the secret that contains the kubeconfig for cluster B:
+   ```bash
+   kubectl create -f - <<EOF
+   apiVersion: rbac.authorization.k8s.io/v1
+   kind: Role
+   metadata:
+     name: remote-cluster-secret-access
+     namespace: <secret-namespace>
+   rules:
+   - apiGroups: [""]
+     resources: ["secrets"]
+     verbs: ["watch", "list", "get"]
+   ---
+   apiVersion: rbac.authorization.k8s.io/v1
+   kind: RoleBinding
+   metadata:
+     name: remote-cluster-secret-access
+     namespace: <secret-namespace>
+   roleRef:
+     apiGroup: rbac.authorization.k8s.io
+     kind: Role
+     name: remote-cluster-secret-access
+   subjects:
+   - kind: ServiceAccount
+     name: calico-typha
+     namespace: calico-system
+   EOF
+   ```
+
+1. Create the RemoteClusterConfiguration in cluster A:
+
+   Within the RemoteClusterConfiguration, we specify the secret used to access cluster B, and the overlay routing mode which toggles the establishment of cross-cluster overlay routes.
+   ```bash
+   kubectl create -f - <<EOF
+   apiVersion: projectcalico.org/v3
+   kind: RemoteClusterConfiguration
+   metadata:
+     name: cluster-b
+   spec:
+     clusterAccessSecret:
+       name: remote-cluster-secret-name
+       namespace: <secret-namespace>
+       kind: Secret
+     syncOptions:
+       overlayRoutingMode: Disabled
+   EOF
+   ```
+
+1. If you have no IP pools in cluster A with NAT-outgoing enabled, skip this step.
+
+   Otherwise, if you have IP pools in cluster A with NAT-outgoing enabled and workloads in that pool will egress to workloads in cluster B, you need to instruct {{prodname}} to not perform NAT on traffic destined for IP pools in cluster B.
```

Comment:
```suggestion
   Otherwise, if you have IP pools in cluster A with NAT-outgoing enabled, and workloads in that pool will egress to workloads in cluster B, you need to instruct {{prodname}} to not perform NAT on traffic destined for IP pools in cluster B.
```